### PR TITLE
[docs][core]Accelerator link

### PR DESCRIPTION
**Pull request by angelinalg**
We mention `ray.util.accelerators` but we don't link to where the types are. Also fixed some style nits.

Fixes https://github.com/anyscale/product/issues/31264

### CODE REVIEW

File: doc/source/ray-core/accelerator-types.rst [hunk 1]
```diff
@@ -8,4 +8,8 @@ Accelerator Types
 Ray supports the following accelerator types:
 
 .. literalinclude:: ../../../python/ray/util/accelerators/accelerators.py
+<<<<<<< HEAD
     :language: python
+=======
```
Comment (angelinalg) [inline doc/source/ray-core/accelerator-types.rst:13]:
```suggestion
```

File: doc/source/ray-core/accelerator-types.rst [hunk 2]
```diff
@@ -8,4 +8,8 @@ Accelerator Types
...
+    :language: python
```
Comment (angelinalg) [inline doc/source/ray-core/accelerator-types.rst:14]:
```suggestion
```

File: doc/source/ray-core/accelerator-types.rst [hunk 3]
```diff
@@ -8,4 +8,8 @@ Accelerator Types
...
+>>>>>>> origin/accelerator-link
```
Comment (angelinalg) [inline doc/source/ray-core/accelerator-types.rst:15]:
```suggestion
```

File: doc/source/ray-core/accelerator-types.rst [hunk 4]
```diff
@@ -8,4 +8,8 @@ Accelerator Types
 Ray supports the following accelerator types:
 
 .. literalinclude:: ../../../python/ray/util/accelerators/accelerators.py
+<<<<<<< HEAD
```
Comment (angelinalg) [inline doc/source/ray-core/accelerator-types.rst:11]:
```suggestion
```

File: doc/source/ray-core/scheduling/accelerators.rst [hunk 5]
```diff
@@ -4,7 +4,7 @@
 Accelerator Support
 ===================
 
-Accelerators (e.g. GPUs) are critical for many machine learning applications.
+Accelerators like GPUs are critical for many machine learning apps.
```
Comment (pcmoritz) [inline doc/source/ray-core/scheduling/accelerators.rst:7]: "machine learning workloads" would be the better term here than "machine learning apps" I think :)

**Review by jjyao — APPROVED** [review]

**Pull request merged**